apps/ui/src/components/sidebar/views/favourites.tsx

```diff
@@ -1,76 +1,91 @@
 import { UnixNanoTimeStamp } from "@graphql-debugger/time";
 
-import { useContext } from "react";
+import { useContext, useState } from "react";
 import { Link, useParams } from "react-router-dom";
 
+import { SearchBox } from "../../../components/search-box";
 import { ClientContext } from "../../../context/client";
 import { Delete } from "../../../icons/delete";
 import { IDS } from "../../../testing";
+import { traceNameIncludes } from "../../../utils/find-traces";
 import { isTraceError } from "../../../utils/is-trace-error";
 import { rootSpanName } from "../../../utils/root-span-name";
 
 export function Favourites() {
   const params = useParams<{ traceId: string }>();
   const { favourites, handleDeleteFavouriteTrace } = useContext(ClientContext);
+  const [searchFavourites, setSearchFavourites] = useState("");
 
   const sortedfavourites = favourites.sort((a, b) => {
```

Comment:
Just caught this. `sortedFavourites` and `filteredFavourites` below should probably wrapped in a `useMemo`.